client-libraries/0.1/how-to-guides/python/check-balance.md

```diff
@@ -0,0 +1,68 @@
+# Check the balance of an address in Python
+
+**The balance of [addresses](root://getting-started/0.1/clients/addresses.md) is kept up to date by all [nodes](root://getting-started/0.1/network/nodes.md) in an IOTA network. To request the balance from a node, you must send it the address whose balance you want to check. In this guide, you check the total balance of addresses that belong to a seed.**
+
+## Packages
+
+To complete this guide, you need to install the following package (if you're using Go modules, you just need to reference this package):
+
+```bash
+pip install pyota
+```
+
+## IOTA network
+
+In this guide, we connect to a node on the [Devnet](root://getting-started/0.1/network/iota-networks.md#devnet).
+
+## Code walkthrough
+
+1. Import the packages
+
+    ```python
+    from iota import Iota
+    ```
+
+2. Define your seed. Replace this seed with one that owns an address with test IOTA tokens
+
+    ```python
+    seed = 'JBN9ZRCOH9YRUGSWIQNZWAIFEZUBDUGTFPVRKXWPAUCEQQFS9NHPQLXCKZKRHVCCUZNF9CZZWKXRZVCWQ'
+    ```
+
+3. Connect to a node
+
+    ```python
+    api = Iota('https://nodes.devnet.iota.org:443', seed)
+    ```
+
+4. Use the [`get_account_data()`](https://github.com/iotaledger/iota.py/blob/master/docs/api.rst#get_account_data) method to ask the node for the current balance of the seed's addresses
+
+    ```python
+    balance = api.get_account_data()
+    print('The balance for your seed is: ', balance['balance'])
+    ```
+
+    :::info:
+    This method starts from index 0 and increments it until an unspent address with no balance is found.
```

Comment:
increments it until n address with no transactions and not spent from is found. it doesn't check for no balance